app/AcquireIntersonArrayBMode.cxx

```diff
@@ -146,13 +147,18 @@ int main( int argc, char * argv[] )
     std::cerr << "Invalid requested depth for probe." << std::endl;
     }
 
+  const double ns = ContainerType::MAX_SAMPLES; // number of samples
+  const double fs = 17000; // [kHz]=[samples/ms] - sampling frequency
+  const double depth = sos * ns / ( 2 * fs );
```

Comment:
Concluding conversations offline, the B-mode uses the setting of the internal scan converter, so we do not need these values in `AcquireIntersonArrayBMode.cxx`.